frontend/src/Components/Helpers/Helpers.js

```diff
@@ -1,5 +1,6 @@
 // Helper functions
 import axios from "axios";
+const _ = require("lodash");
```

Comment:
use es6 `import` instead. Also, import only the function you need (orderBy) instead of the whole lodash module, which is quite heavy/large.

Reply:
Fixed this issue, and only have lodash.orderby in package.json, but all the other lodash modules are still appearing in yarn.lock. I tried  yarn remove, but that doesn't work. Any suggestions on removing these?

Reply:
Hummm, that's weird. Generally yarn should be able to resolve this....
Not ideal, but you could nuke the lock file and re-gen it with yarn install

Reply:
it appears that these modules are already on the master yarn.lock file. I tried the alternate of deleting and regenerating it.. but they are still in the yarn.lock file, despite not being in the package.json. Is it ok to go ahead and. merge?

Reply:
yep, should be gtg